src/modules/filesystem/NativeFile.h

```diff
@@ -25,6 +25,9 @@
 // C
 #include <cstdio>
 
+// SDL
+#include <SDL3/SDL_iostream.h>
```

Comment:
Can this include be moved to `NativeFile.cpp`? And then this can be a `typedef struct SDL_IOStream SDL_IOStream;` or something - I'd just like to minimize how much of SDL is in public module headers when possible.